apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -191,7 +191,9 @@ export class TaskCardUc {
 	}
 
 	private checkSchoolYearEndDate(schoolYearEndDate: Date, dueDate: Date) {
-		if (schoolYearEndDate < dueDate) {
+		const schoolYearEndDateAtMidnight = new Date(schoolYearEndDate).setHours(0, 0, 0, 0);
```

Comment:
just a note: this would compare last midnight (to set it to next midnight, it would be 24,0,0,0)
but it still works, because you do it for both
also, sometimes you need to use getTime(), to not affect the original date, but I think is not needed here

Reply:
Okay, I didn't know. Thanks. But as you said. Important is to have the exact same time here.